ValidateOtp/handler.ts

```diff
@@ -0,0 +1,123 @@
+import * as express from "express";
+
+import { Context } from "@azure/functions";
+import { NonNegativeInteger } from "@pagopa/ts-commons/lib/numbers";
+import {
+  IResponseErrorNotFound,
+  ResponseErrorInternal,
+  ResponseErrorNotFound
+} from "@pagopa/ts-commons/lib/responses";
+import { FiscalCode } from "@pagopa/ts-commons/lib/strings";
+import { toError, tryCatch2v } from "fp-ts/lib/Either";
+import { identity } from "fp-ts/lib/function";
+import { none, Option, some } from "fp-ts/lib/Option";
+import {
+  fromEither,
+  fromLeft,
+  TaskEither,
+  taskEither
+} from "fp-ts/lib/TaskEither";
+import { ContextMiddleware } from "io-functions-commons/dist/src/utils/middlewares/context_middleware";
+import { RequiredBodyPayloadMiddleware } from "io-functions-commons/dist/src/utils/middlewares/required_body_payload";
+import { RequiredParamMiddleware } from "io-functions-commons/dist/src/utils/middlewares/required_param";
+import {
+  withRequestMiddlewares,
+  wrapRequestHandler
+} from "io-functions-commons/dist/src/utils/request_middleware";
+import * as t from "io-ts";
+import {
+  IResponseErrorForbiddenNotAuthorized,
+  IResponseErrorInternal,
+  IResponseSuccessJson,
+  ResponseSuccessJson
+} from "italia-ts-commons/lib/responses";
+import { RedisClient } from "redis";
+import { Otp } from "../generated/definitions/Otp";
+import { OtpCode } from "../generated/definitions/OtpCode";
+import { OtpValidationResponse } from "../generated/definitions/OtpValidationResponse";
+import { Timestamp } from "../generated/definitions/Timestamp";
+import { ValidateOtpPayload } from "../generated/definitions/ValidateOtpPayload";
+import { getTask } from "../utils/redis_storage";
+
+const OTP_PREFIX = "OTP_";
```

Comment:
This prefix is shared with `io-functions-cgn`. We should find a way to share this value for example we can add it into `io-functions-commons`.

Reply:
I'm wondering about place this properties into io-fn-commons. Maybe we can externalize them into env variables?

Reply:
This value cannot change over time, is a constant value. If we change that value we broke the function (is needed a data migration). IMHO is more solid to take that value from the same place, the "model" should be shared from all the functions that access the database.

Reply:
OK I'll make a PR

Reply:
@balanza suggest adding comments over these constants definitions ([comment](https://github.com/pagopa/io-functions-commons/pull/135#issuecomment-799228950))